selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -568,11 +557,10 @@ public void waitFolderDefinedTypeOfFolderByPath(String pathToFolder, String type
    * @param pathToItem full path to item in the codenvy project explorer
    */
   public void waitRemoveItemsByPath(String pathToItem) {
-    String locator = "//div[@path='/" + pathToItem + "']/div";
-    new WebDriverWait(seleniumWebDriver, 10)
-        .until(
-            ExpectedConditions.not(
-                ExpectedConditions.presenceOfAllElementsLocatedBy(By.xpath(locator))));
+    loadPageTimeout.until(
+        ExpectedConditions.not(
```

Comment:
Let's use static import for simplifying static method references similar to how it had been done in other parts of code.